internal/services/postgres/postgresql_database_resource.go

```diff
@@ -5,6 +5,8 @@ import (
 	"log"
 	"time"
 
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/postgres/migration"
+
```

Comment:
looks like there's an extra space here

Reply:
Thanks @catriona-m. I thought there was some convention here - https://github.com/hashicorp/terraform-provider-azurerm/blob/620839508fa6f8a0e46c6129b94b64a42aa00b55/internal/services/postgres/postgresql_server_resource.go#L10-L13

Fixed in 0c87b1c (will squash afterwards). Is there a linter for that, btw?

Reply:
there is: `make goimports` is what we should go by